frontend/pages/index.js

```diff
@@ -0,0 +1,77 @@
+import { useState } from 'react';
+import Head from 'next/head';
+import axios from 'axios';
+
+import { Textarea, Text, Button, Box } from '@chakra-ui/core';
+
+export default function Home() {
+  const [text, setText] = useState('');
+  const [isSubmitting, setIsSubmitting] = useState(false);
+  const [analyzeResult, setAnalyzeResult] = useState(null);
+
+  const handleTextChange = (e) => {
+    setText(e.target.value);
+    setAnalyzeResult(null);
+  };
+
+  const handleOnClick = () => {
+    setIsSubmitting(true);
+
+    // TODO: add the correct endpoint.
+    axios
+      .post('/analyze', {
+        text,
+      })
+      .then(function (response) {
+        // assumes that response is a number between 0.0-1.0
+        setAnalyzeResult(response.data.result);
```

Comment:
assumes the response will just be a `0.0-1.0`.

(the response object should look like:
```
{
  result: 0.5
}
```
(json format)

Reply:
The current response that comes out of the api is the message ("the model thinks that..") with the resulting probability embedded in it, see https://github.com/oren0e/fake-news/blob/bd2387cd4fdfd1a8a9d2c90a0bf8100c4fe89ad9/backend/ft_model.py#L55

I can change it so that the above `predict()` method will return only the numerical result. What do you think is better?

Reply:
I think its better to let the frontend take care of the text, and so we keep the payload pretty small.

Reply:
ok agreed